Formula/vitess.rb

```diff
@@ -0,0 +1,57 @@
+class Vitess < Formula
+  desc "Database clustering system for horizontal scaling of MySQL"
+  homepage "http://vitess.io"
+
+  url "https://github.com/arthurnn/vitess.git",
+      :revision => "75b5eb418f2b5d5ac1e8f5f8fd6bad723d0dcd48",
+      :branch => "arthurnn/easier_build"
+  version "2.1.1.75b5eb41"
+  head "https://github.com/arthurnn/vitess.git"
```

Comment:
Will need to wait until we have an official upstream tag before this gets merged but can review here before that.

Reply:
Only CI failures relate to that too: https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/19280/version=high_sierra/testReport/junit/brew-test-bot/high_sierra/audit_vitess___online___new_formula/